pkg/api/metrics.go

```diff
@@ -213,6 +222,10 @@ func GetTestDataRandomWalk(c *models.ReqContext) Response {
 
 	resp, err := tsdb.HandleRequest(context.Background(), dsInfo, request)
 	if err != nil {
+		if errors.Is(err, tsdberrs.BadRequest{}) {
```

Comment:
I cannot see any code referring to creating a `tsdberrs.BadRequest` so this should probably be removed.